### PR TITLE
feat: Adding ClientNetworkAnimator to samples utilities package [MTT-5025]

### PR DESCRIPTION
**Pull request by fernando-cortez**
### Description
This PR adds ClientNetworkAnimator to the samples utilities package.

### Issue Number(s)
[MTT-5025](https://jira.unity3d.com/browse/MTT-5025).

### Contribution checklist
 - [ N/A] Tests have been added for boss room and/or utilities pack
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink
 - [x] An Index entry has been added in readme.md if applicable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ## [unreleased] - aaaa-mm-dd
 
 ### Added
-*
+* ClientNetworkAnimator component has been added to the Samples Utilities Package. This allows for authority on Animators to be passed onto clients, meaning animations will be client-driven.
```
Comment (SamuelBellomo) [inline CHANGELOG.md:12]: There's a changelog file directly in the utilities package, would you mind moving this there?

**Review by SamuelBellomo — APPROVED** [review]

File: Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Multiplayer Samples Co-op Changelog
 
+## [unreleased] - aaaa-mm-dd
```
Comment (SamuelBellomo) [inline Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md:3]: a for año?
:P

**Review by SamuelBellomo — APPROVED** [review]

File: Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md (hunk 2 above)
Comment (SamuelBellomo) [reply Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md:3]: you can resolve this, this isn't a merge blocker :P

Comment (fernando-cortez) [reply Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md:3]: I took this from the Boss Room changelog 😂 It might have been haha.

**Pull request merged**